package.json

```diff
@@ -32,14 +32,16 @@
   "dependencies": {
     "@babel/code-frame": "^7.15.8",
     "chalk": "^4.1.2",
-    "create-jest-runner": "^0.11.0",
-    "tsd-lite": "^0.5.0"
+    "create-jest-runner": "^0.11.1",
+    "tsd-lite": "^0.5.4"
```

Comment:
This is where I bumped into missing types of `create-jest-runner`. Thanks for quick fix!

Had to expose one type from `tsd-lite` as well. Some fixes on the way (;

Reply:
Exciting stuff 😃